python/dgl/graphbolt/base.py

```diff
@@ -108,22 +108,38 @@ class CopyTo(IterDataPipe):
 
     Functional name: :obj:`copy_to`.
 
-    This is equivalent to
+    When `data` has `to` method implemented, `CopyTo` will be equivalent to
 
     .. code:: python
 
        for data in datapipe:
            yield data.to(device)
 
+    For `gb.MiniBatch`, only a part of attributes will be transferred to
+    accelerate the process:
+
+    - When `seed_nodes` is not None and `node_pairs` is None, node related
+    task is inferred. Only `labels`, `sampled_subgraphs`, `node_features`
+    and `edge_features` will be transferred.
+
+    - When `node_pairs` is not None and `seed_nodes` is None, edge/link
+    related task is inferred. Only `labels`, `compacted_node_pairs`,
+    `compacted_negative_srcs`, `compacted_negative_dsts`, `sampled_subgraphs`,
+    `node_features` and `edge_features` will be transferred.
+
+    - Otherwise, all attributes will be transferred. If you want some other
+    attributes to be transferred as well, please indicate the name in the
+    `extra_attrs`.
```

Comment:
add an detailed example to showcase how to copy extra attributes such as `input_nodes` via `extra_attributes`.